### ISSUE TITLE
can not update fastlane because of mini_magick dependency

### ISSUE THREAD

**Issue by ottosuess** [KrauseFx/fastlane-plugin-appicon#50]
to be honest i now nothing about bundler, but i suspect this message is trying to tell me it can't update fastlane to the latest version because of conflicts in the required mini_magick dependency?

```
Bundler could not find compatible versions for gem "mini_magick":
  In Gemfile:
    fastlane (~> 2.127.2) was resolved to 2.127.2, which depends on
      mini_magick (< 5.0.0, >= 4.9.4)

    fastlane-plugin-appicon was resolved to 0.14.0, which depends on
      mini_magick (~> 4.5.1)
```

mini_magick 4.5.1 has a potential security vulnerability.

**Comment by joshdholtz** [comment]
@ottosuess Whoops! I will bump the version on this and get a new build out right now 😊 Stay tuned and thanks for reporting ❤️

**Comment by joshdholtz** [comment]
@ottosuess A fix for this is released in `0.14.1`🚀 

I was not able to make release for this since github is 500ing when viewing/making pull requests but this will be committed into this code base when that's back up 🙃
